src/client/components/navBar/navBar.css

```diff
@@ -0,0 +1,30 @@
+.navBar {
+  display: flex;
+  align-items: center;
+  padding-left: 30%;
+  width: 100%;
+  background-color: #34414D;
+  color: white;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+a:hover {
```

Comment:
`.nav a:hover `

Reply:
After applying this fix & the one above, it might be worth reordering this file so similar rules are grouped